src/Orleans.Core/Async/TaskExtensions.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Orleans
 {
-    internal static class OrleansTaskExtentions
+    public static class OrleansTaskExtentions
```

Comment:
Likewise here. I'd prefer we namespace these to something internal at least, since otherwise we can potentially introduce some clashes